julia/README.md

```diff
@@ -87,6 +87,8 @@ julia> typeof(convert(Int64, m.x))
 Int64
 ```
 
+The inverse of `juliaize` is `dexize`, it is currently very limited.
```

Comment:
I haven't given it proper docs yet, because it only does Float32.
It's mostly just for testing purposes.
We need a proper API for this.

Reply:
Sure, we can extend it if you'd like. Just let me know what would be helpful to have.

Reply:
I guess it is something like `create_literal` that works like `insert` except instead of taking an `Atom` it takes a C compatible value for a Int/Float32/Float64/Array.
Possibly `ctypes` doesn't allow that directly, so maybe it needs to be wrapped into a tagged union?
I guess maybe accepting the same tagged union that comes out of the atom's pointer makes sense.
(Except right now that doesn't support arrays)

Reply:
Wouldn't a function that converts a `CAtom` into an `Atom` be sufficient? That's what I would imagine. And then we can add more cases to `CAtom` if that would be useful.

Reply:
Cool, which cases would you like to have? I'm happy to add them for you

Reply:
`Float32`, `Float64` (particularly since can't input those as literals https://github.com/google-research/dex-lang/issues/497)
Arrays would be nice, but given we can't currently convert `Atom` to `CAtom` for arrays anyway, that doesn't matter so much.
Integer types would be nice, for completeness, but not particularly interesting for AD.

Reply:
Added that in #657